### PR TITLE
Drop support for Python < 3.7

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -4,21 +4,19 @@
...
     py311
-    pypy
     pypy3
     docs
     coverage
 
 [testenv]
 usedevelop = true
+package = wheel
+wheel_build_env = .pkg
```
Comment (dataflake) [inline tox.ini:19]: Out of curiosity, what's that for?

**Review by dataflake — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (icemac) [reply tox.ini:19]: It is taken from https://hynek.me/articles/turbo-charge-tox/ to speed up build time.
On my local machine before adding it a tox run of a single Python version took 4.3 s now it takes 1.1 s.

Comment (icemac) [reply tox.ini:19]: See also https://github.com/zopefoundation/meta/pull/201

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

File: tox.ini (hunk 1 above)
Comment (dataflake) [reply tox.ini:19]: 😱  Are you adding that to meta/config?

**Pull request merged**